src/ifcopenshell-python/ifcopenshell/api/alignment/__init__.py

```diff
@@ -0,0 +1,24 @@
+# IfcOpenShell - IFC toolkit and geometry engine
+# Copyright (C) 2022 Dion Moult <dion@thinkmoult.com>
+#
+# This file is part of IfcOpenShell.
+#
+# IfcOpenShell is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# IfcOpenShell is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with IfcOpenShell.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Manages alignment and alignment geometry
+"""
+
+from .alignment import *
+from .segments import *
```

Comment:
The rest of the API typically has functions within each module. It may be worthwhile breaking up alignment.py and segments.py into (verb) functions, one per file like everything else. Each file would then have its corresponding test cases mirrored in the test folder.

Reply:
Good advice. I've been doing just that. Here is the WIP https://github.com/IfcOpenShell/IfcOpenShell/tree/RAB_alignment_api/